llvm/lib/Target/AIE/AIEBaseSubtarget.cpp

```diff
@@ -381,6 +383,26 @@ class PropagateIncomingLatencies : public ScheduleDAGMutation {
           }))
         continue;
 
+      // Do not change the latency if the REG_SEQUENCE has one source
```

Comment:
Humm, this is exactly the situation that I had in mind when I looked to `MacroFusion`, place `REG_SEQUENCE` near to the user. Great!